releasenotes/notes/csm-7-50-14413f70a64656f9.yaml

```diff
@@ -0,0 +1,13 @@
+---
+features:
+  - |
+    CWS: Ability to send a signal to a process when a rule was triggered.
+    CWS: Add Kubernetes user session context to events, in particular the username, UID and groups of the user that ran the commands remotely.
+
+enhancements:
+  - |
+    CWS: Support for Ubuntu 23.10.
+    CWS: Support using TCP in addition in UNIX sockets.
```

Comment:
in addition "to" maybe ? and maybe explain a bit what we mean by support ? I don't think it's super clear

Reply:
Indeed. I completely removed it from the change log